README.md

```diff
@@ -86,3 +86,21 @@ Dynamic forms allow you to make a form without having to register it before the
     state:show()
 
 Make sure you call state:show()
+
+
+## Formspec on nodes
+SmartFS is able to attach a formspec to a node metadata. The attached form can be opened by any player and all players see the same. If a player enter information and take updates on the form, all users see the updated form instantly. All changes are saved in node meta data and visible to all players, to show only a single player info please show them another form using form:show.
+There is a "reset" implemented if all players leave the node formspec, the initial state is restored. The parameters are implemented persistant so if you need to store sometging to node just use state:setparam() / state:getparam(). But do not save sensitive data since the nodemeta is sent to all player clients.
```

Comment:
something

Also, you should wrap the text manually (the lines shouldn't be longer than 80 characters)